<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>271</width>
     <height>292</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="no_masukLabel">
      <property name="text">
       <string>no_masuk</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edit_no_masuk"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="kd_zakatLabel">
      <property name="text">
       <string>kd_zakat</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edit_kd_zakat"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="kd_muzakiLabel">
      <property name="text">
       <string>kd_muzaki</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edit_kd_muzaki"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="jmlh_masukLabel">
      <property name="text">
       <string>jmlh_masuk</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edit_jmlh_masuk"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="bentukLabel">
      <property name="text">
       <string>bentuk</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edit_bentuk"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="tgl_masukLabel">
      <property name="text">
       <string>tgl_masuk</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="edit_tgl_masuk"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="norekLabel">
      <property name="text">
       <string>norek</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="edit_norek"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="ketLabel">
      <property name="text">
       <string>ket</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="edit_ket"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>status</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="edit_status"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FORM ZAKAT MASUK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCetak">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cetak</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QTableView" name="tabel_Zakatmasuk">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>361</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
